dev-util/intel-graphics-compiler/intel-graphics-compiler-1.0.9636.ebuild

```diff
@@ -0,0 +1,112 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+CMAKE_BUILD_TYPE="Release"
+LLVM_MAX_SLOT="13"
+MY_PN="igc"
+MY_P="${MY_PN}-${PV}"
+#PYTHON_COMPAT=( python3_{8..10} )
+PYTHON_COMPAT=( python3_10 )
+
+inherit cmake flag-o-matic llvm python-any-r1 git-r3
+
+DESCRIPTION="LLVM-based OpenCL compiler for OpenCL targetting Intel Gen graphics hardware"
+HOMEPAGE="https://github.com/intel/intel-graphics-compiler"
+SRC_URI="https://github.com/intel/${PN}/archive/${MY_P}.tar.gz -> ${P}.tar.gz"
+EGIT_REPO_URI="https://github.com/KhronosGroup/SPIRV-Tools.git"
```

Comment:
This is wrong. Using vcs in a non 999 ebuild is not correct. Please point to snapshot archives. Could you explain further why those are needed?

Reply:
My knowledge about ebuild system is rather poor, so my first goal is to make it compilable. Probably it could be better done, but sorry I do not how.

Reply:
Github has the ability to download any commit as a tarball, so you can replace the git stuff with `SRC_URI="https://github.com/example/example/archive/${COMMIT}.tar.gz -> name-${P}.tar.gz`